java-client-generated/src/main/java/org/ergoplatform/explorer/client/model/AdditionalRegister.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Ergo Explorer API v1
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * OpenAPI spec version: 1.0
+ * 
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+package org.ergoplatform.explorer.client.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Objects;
+
+/**
+ * AdditionalRegisters
+ */
+
+
+public class AdditionalRegister {
+
+    @SerializedName("serializedValue")
+    public String serializedValue;
+    @SerializedName("sigmaType")
+    public String sigmaType;
+    @SerializedName("renderedValue")
+    public String renderedValue;
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    return serializedValue.equals(((AdditionalRegister) o).serializedValue);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(super.hashCode());
+  }
```

Comment:
The equality implementation is weird.

Reply:
To be honest I just copied what the other generated classes had. We can remove it completely when it is me.